diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -123,6 +124,16 @@ data class CommonConfiguration(private val configuration: Map<String, String>?)
         configuration?.get("disabledChapters")
     }
 
+    /**
+     * Get version of kotlin from configuration
+     */
+    val kotlinVersion: KotlinVersion by lazy {
+        configuration?.get("kotlinVersion")?.kotlinVersion() ?: run {
+            log.error("Kotlin version not specified. Will be use current version")
```

Comment:
Please log which version is current here. It's version of kotlin-compiler embedded in diktat, not current user's project version.